firebase-sessions/CHANGELOG.md

```diff
@@ -1,7 +1,15 @@
 # Unreleased
+
+
+# 2.0.7
 * [fixed] Removed extraneous logs that risk leaking internal identifiers.
 
 
+## Kotlin
```

Comment:
Sessions shouldn't have this section.

Reply:
Updated and fixed from the last release as well